velox/buffer/tests/BufferTest.cpp

```diff
@@ -41,7 +41,8 @@ static_assert(!Buffer::is_pod_like_v<std::shared_ptr<int>>, "");
 class BufferTest : public testing::Test {
  protected:
   void SetUp() override {
-    pool_ = memoryManager_.getChild();
+    pool_ =
+        memoryManager_.getPool("BufferTest", memory::MemoryPool::Kind::kLeaf);
```

Comment:
nit: Would it make sense to add getLeafPool method?

```
memoryManager_.getLeafPool("BufferTest");
```

Reply:
As discussed offline, we will leave API naming changes in followup.